src/core/services/gesture/gesture.js

```diff
@@ -302,7 +302,7 @@ angular.module('material.core')
     onCancel: angular.noop,
     options: {},
 
-    dispatchEvent: dispatchEvent,
+    dispatchEvent: typeof jQuery !== 'undefined' && angular.element === jQuery ? jQueryDispatchEvent : nativeDispatchEvent,
```

Comment:
You are right, jQuery may be defined as another name in global namespace, and then properly register using the new `ngJq` directive.

Maybe we should only check if `angular.element` has `trigger` function ? jqLite doesn't have this function.

https://github.com/angular/angular.js/blob/master/src/Angular.js#L1532-L1536